src/Twilio/Rest/Authy/V1/Service/Entity/Factor/ChallengeOptions.cs

```diff
@@ -43,6 +43,10 @@ public class CreateChallengeOptions : IOptions<ChallengeResource>
         /// Hidden details provided to contextualize the Challenge
         /// </summary>
         public string HiddenDetails { get; set; }
+        /// <summary>
+        /// The Twilio-Authy-Sandbox-Mode HTTP request header
+        /// </summary>
+        public string TwilioAuthySandboxMode { get; }
```

Comment:
Don't think the head support stuff belongs in this PR.

Reply:
I know... main issue is that these are being rendered because of shared infrastructure that was merged from the other yoyodyne PR. I'm not sure how I'd go about rolling back those changes for the libs that don't yet have header support.